wallet/src/de/schildbach/wallet/Constants.java

```diff
@@ -138,8 +137,12 @@ public final static class Files {
         /** Filename of the block checkpoints file. */
         public static final String CHECKPOINTS_FILENAME = "checkpoints" + FILENAME_NETWORK_SUFFIX + ".txt";
 
+        /** Filename of the bootstrap masternode list diff file. */
         public static final String MNLIST_BOOTSTRAP_FILENAME = "mnlistdiff" + FILENAME_NETWORK_SUFFIX + ".dat";
 
+        /** Filename of the bootstrap qrinfo file. */
+        public static final String QRINFO_BOOTSTRAP_FILENAME = "qrinfo" + FILENAME_NETWORK_SUFFIX + ".dat";
+
```

Comment:
This naming system is not ideal because all of these files are stored in the assets folder and all builds will include asset files that are only used by other builds.

Reply:
In a future PR we can re-organize the assets folder and its usage in build.grade.

All builds need some files (BIP39 word lists). Some builds only need particular files (checkpoints, mnlistdiff, etc).